lib/commons/color/get-background-color.js

```diff
@@ -53,17 +53,17 @@ function getBgColor(elm, elmStyle) {
  * @param {Element} bgNode
  * @return {Boolean}
  */
-function contentOverlapping(targetElement, bgNode) {
+function contentOverlapping(targetElement, fgNode) {
 	// get content box of target element
-	// check to see if the current bgNode is overlapping
-	var targetRect = targetElement.getClientRects()[0];
```

Comment:
FYI: I used `getClientRects` because it would target the diff in coverage for an inline label wrapping an input, as opposed to the outer bounding box of the label. I guess with this change, the `fgNode` is the input instead of the label in that case?